Documentation/ceph-nfs-crd.md

```diff
@@ -4,19 +4,30 @@ weight: 3100
 indent: true
 ---
 
-# Ceph NFS Gateway CRD
+# Ceph NFS Server CRD
 
 ## Overview
 
-Rook allows exporting NFS shares of the filesystem or object store through the CephNFS custom resource definition. This will spin up a cluster of [NFS Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) servers that coordinate with one another via shared RADOS objects. The servers will be configured for NFSv4.1+ access, as serving earlier protocols can inhibit responsiveness after a server restart.
+> **WARNING**: We do not recommend using NFS in Ceph v16.2.0 through v16.2.6. If you are using Ceph
+> v15, please wait to upgrade until v16.2.7 is released.
+> [Upgrade steps are outlined below.](#upgrading-from-ceph-v15-to-v16)
 
-## Samples
+Rook allows exporting NFS shares of the filesystem or object store through the CephNFS custom
+resource definition. This will spin up a cluster of
+[NFS Ganesha](https://github.com/nfs-ganesha/nfs-ganesha) servers that coordinate with one another
+via shared RADOS objects. The servers will be configured for NFSv4.1+ access only, as serving
+earlier protocols can inhibit responsiveness after a server restart.
 
-The following sample will create a two-node active-active cluster of NFS Ganesha gateways. The recovery objects are stored in a RADOS pool named `myfs-data0` with a RADOS namespace of `nfs-ns`.
+## Samples
+The following sample assumes Ceph v15 and will create a two-node active-active cluster of NFS
```

Comment:
Are we actually assuming v16.2.7 now, instead of v15?

Reply:
I can change this and the example manifest I think